config/seed-peer-generator/src/main.rs

```diff
@@ -57,9 +57,7 @@ fn get_validator_set(endpoint: String, peer_id: PeerId) -> anyhow::Result<Option
 /// Convert ValidatorInfo to a seed peer
 fn to_seed_peer(validator_info: &ValidatorInfo) -> (PeerId, Vec<NetworkAddress>) {
     let peer_id = *validator_info.account_address();
-    let seed_pubkey = validator_info.network_identity_public_key();
     let network_addr =
         NetworkAddress::try_from(&validator_info.config().full_node_network_address).unwrap();
-    let seed_addr = network_addr.append_prod_protos(seed_pubkey, HANDSHAKE_VERSION);
-    (peer_id, vec![seed_addr])
+    (peer_id, vec![network_addr])
```

Comment:
we should log addresses that fail to decode and just not include them in the seed_peers list

this fn can log errs and just return `Option<(PeerId, Vec<NetworkAddress>)>`

then above replace `map` with `filter_map` to ignore `None`s

Reply:
Done